CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# CHANGELOG
+
+## 0.0.1
```

Comment:
Its a "new feature" so imho it should be 0.1.0

Reply:
It was `null.null.null` before this release, so we even bumped major ;)

Reply:
I am also fine with that if this is already our defining API ;-)